test/bufr_ncep_satwind_avhrr.yaml

```diff
@@ -0,0 +1,186 @@
+#(C) Copyright 2021 NOAA/NWS/NCEP/EMC
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# SAID, CLAT, CLON, YEAR, MNTH, DAYS, HOUR, MINU, SECO, SWCM, SAZA, OGCE, SCCF, SWQM
+# HAMD, PRLC, WDIR, WSPD
+# GNAP, PCCF
+
+observations:
+  - obs space:
+      name: bufr
+      #obsdatain: "./testinput/bufr_read_2_dim_blocks.bufr"
+      obsdatain: "./testinput/bufr_ncep_satwind_avhrr.bufr_d"
+
+      exports:
+        variables:
+          satelliteIdentifier:
+            query: "*/SAID"
+
+          latitude:
+            query: "*/CLAT"
+
+          longitude:
+            query: "*/CLON"
+
+          year:
+            query: "*/YEAR"
+
+          month:
+            query: "*/MNTH"
+
+          day:
+            query: "*/DAYS"
+
+          hour:
+            query: "*/HOUR"
+
+          minute:
+            query: "*/MINU"
+
+          second:
+            query: "*/SECO"
+
```

Comment:
Remove these variables.
And, add the following:
```
          timestamp:
            datetime:
              year: "*/YEAR[1]"
              month: "*/MNTH[1]"
              day: "*/DAYS[1]"
              hour: "*/HOUR[1]"
              minute: "*/MINU[1]"
```

Reply:
The yaml has now been changed to match the above. Will be uploaded shortly.

Reply:
Actually, I had a question on this edit;  Is "seconds" no longer included?  The update only includes down to minutes.

Reply:
Seconds are optional. Include if they are available (not all BUFR files have seconds)

Reply:
```
          heightAssignmentMethod:
            query: "*/HAMD[1]"

          pressure:
            query: "*/PRLC[1]"

          generatingApplication:
            query: "*/GQCPRMS/GNAP[1]"

          percentConfidence:
            query: "*/GQCPRMS/PCCF[1]"
```

Those indices on HAMD and PRLC should not be needed at all (but should'nt hurt either) investigating.